### ISSUE TITLE
Sorting Navigates to Parent Category (Product Listing) [NitroSell]

### ISSUE THREAD

**Issue by DerekFoulk** [DerekFoulk/nwca-nitrosell#51]
If you use the menu to navigate to 'HDMI (Cables)' and sort by 'Price (Ascending)', you are apparently taken to 'Video Cables' (and presumably shown *those* listing in the said order...

Device: Samsung Galaxy S6
Browser: Google Chrome

**Comment by DerekFoulk** [comment]
Duplicated from the NitroSell ticketing system "GoLive - Price Sort on Search.asp pages - ineffective" -
---
I ran into an issue last night regarding the sort dropdown... Please follow these steps to recreate:
- Navigate to [our homepage](http://ws4301-4323.staging.nitrosell.com/)
- Use our main navigation menu to go to 'HDMI > HDMI'
- This should take you to http://.../Cables-%26-Adapters-Video-Cables/?subcategory=HDMI
- Please take note of the breadcrumbs at the top of the page that read:
"Home / Cables & Adapters / Video Cables / HDMI"
- Also note that only HDMI cables and accessories are shown in the results/listings
- Now change the sort dropdown from what it is currently on to something else like 'Price (Ascending)'
- Take a look at the breadcrumbs again, and the results... We are now viewing the entire 'Video Cables' category instead of its 'HDMI' subcategory, and there are things like RCA cables and various other non-HDMI products listed

So to summarize- changing the value of the "Sort" dropdown while viewing a 'SubCategory' will navigate you away from that 'SubCategory' and take you to the parent 'Category'. 

*Please note: this happens whenever you are in a subcategory. It does not matter which one you are looking at... Changing the sort dropdown results in the user being taken up a level in the navigation, thus seeing results from many other subcategories instead of the desired subcategory results.

**Comment by DerekFoulk** [comment]
See ticket for more information:
https://portal.nitrosell.com/support/showticket/?ticket_id=101570

> **Brendan Quigley of NitroSell Limited said:**

> Hi Derek,

> The sort on search pages will now respect the sorting chosen by the customer.

> Regards,
> Brendan

> *24 days ago, on Tuesday, 9th June, 2015, 09:13 UTC*

> **Brendan Quigley of NitroSell Limited said:**

---

> Hi Derek,

> This didn't need a template update.

> You can add extra fields to the dropdown by adding to

> var arrSortFields = {
> "name" : "Name",
> "price" : "Price"
> };

> from the following list:

> name, price, price2 (webprice), code, brand, theme, subcategory,
> subdescription1, subdescription2, subdescription3, customtext1,
> customtext2, customtext3, customtext4, customtext5

> If you update the script to remove the ?sort parameter the results will sort by relevance.

> Regards,
> Brendan

> *23 days ago, on Wednesday, 10th June, 2015, 09:45 UTC*

**Comment by DerekFoulk** [comment]
This has been resolved for the most part. All that is left to do is add the extra sort fields to the dropdown as shown in the previous comment...

**Comment by DerekFoulk** [comment]
Extra sort options were added in 447dfdf1942b35274ed4be64a46b394e0d6033c9.
